src/Goerli-DssSpell.t.base.sol

```diff
@@ -989,6 +1052,106 @@ contract GoerliDssSpellTestBase is Config, DSTest, DSMath {
         assertEq(token.balanceOf(address(join)), 0);
     }
 
+    function getSignatures(bytes32 signHash) internal returns (bytes memory signatures, address[] memory signers) {
+        // seeds chosen s.t. corresponding addresses are in ascending order
+        uint8[30] memory seeds = [8,10,6,2,9,15,14,20,7,29,24,13,12,25,16,26,21,22,0,18,17,27,3,28,23,19,4,5,1,11];
+        uint numSigners = seeds.length;
+        signers = new address[](numSigners);
+        for(uint i; i < numSigners; i++) {
+            uint sk = uint(keccak256(abi.encode(seeds[i])));
+            signers[i] = hevm.addr(sk);
+            (uint8 v, bytes32 r, bytes32 s) = hevm.sign(sk, signHash);
+            signatures = abi.encodePacked(signatures, r, s, v);
+        }
+        assertEq(signatures.length, numSigners * 65);
+    }
+
+    function addressToBytes32(address addr) internal pure returns (bytes32) {
+        return bytes32(uint256(uint160(addr)));
+    }
+
+    function oracleAuthRequestMint(
+        bytes32 sourceDomain,
+        bytes32 targetDomain,
+        uint256 toMint,
+        uint256 expectedFee
+    ) internal {
+        TeleportOracleAuthLike oracleAuth = TeleportOracleAuthLike(chainLog.getAddress("MCD_ORACLE_AUTH_TELEPORT_FW_A"));
+        giveAuth(address(oracleAuth), address(this));
+        (bytes memory signatures, address[] memory signers) = getSignatures(oracleAuth.getSignHash(TeleportGUID({
+            sourceDomain: sourceDomain,
+            targetDomain: targetDomain,
+            receiver: bytes32(uint256(uint160(address(this)))),
+            operator: bytes32(0),
+            amount: uint128(toMint),
+            nonce: 1,
+            timestamp: uint48(block.timestamp)
+        })));
+        oracleAuth.addSigners(signers);
+        oracleAuth.requestMint(TeleportGUID({
+            sourceDomain: sourceDomain,
+            targetDomain: targetDomain,
+            receiver: bytes32(uint256(uint160(address(this)))),
+            operator: bytes32(0),
+            amount: uint128(toMint),
+            nonce: 1,
+            timestamp: uint48(block.timestamp)
+        }), signatures, expectedFee * WAD / toMint, 0);
+    }
+
+    // NOTE: Only executable by forge
+    function checkTeleportFWIntegration(
+        bytes32 sourceDomain,
+        bytes32 targetDomain,
+        uint256 line,
+        address fee,
+        address gateway,
+        address escrow,
+        uint256 toMint,
+        uint256 expectedFee
+    ) public {
+        TeleportJoinLike join = TeleportJoinLike(chainLog.getAddress("MCD_JOIN_TELEPORT_FW_A"));
+        TeleportRouterLike router = TeleportRouterLike(chainLog.getAddress("MCD_ROUTER_TELEPORT_FW_A"));
```

Comment:
Both are fine but I guess the `addr.addr(...)` is shorter and what we've used in other places:
```suggestion
        TeleportJoinLike join = TeleportJoinLike(addr.addr("MCD_JOIN_TELEPORT_FW_A"));
        TeleportRouterLike router = TeleportRouterLike(addr.addr("MCD_ROUTER_TELEPORT_FW_A"));
```